ministryofjustice/correspondence_tool_staff
# CT-3052 Added the buttons for assigning/reassigning responders

## Description

Add 2 functions for assigning a case to a person in Branston team
1 - Assign a case to a responder  ---- this is hardly testable from UI due to the self-signed process after the creation of a complaint case through UI, but it would be useful for all those opening cases migrated from Access DB.  This feature should have been fully covered by those tests including feature tests. 
2 - Reassign a case to another responder

Also some changes have been made to the message in case history related to those actions

## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3052
### Deployment


### Manual testing instructions

1. Assign to a team member if there is no assignment yet --> this scenario is difficult to create

2. Reassign to another team member
- Create a offender sar case
- Click button 'Change team member'
- choose one of member
- Confirm it and check the case history
